src/nunit.analyzers/ConstActualValueUsageAnalyzer/ConstActualValueUsageAnalyzer.cs

```diff
@@ -0,0 +1,64 @@
+using System.Collections.Immutable;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+using NUnit.Analyzers.Constants;
+using NUnit.Analyzers.Extensions;
+
+namespace NUnit.Analyzers.ConstActualValueUsage
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    public class ActualConstValueUsageAnalyzer : DiagnosticAnalyzer
```

Comment:
I think we should rename the class to follow the file name.
```suggestion
    public class ConstActualValueUsageAnalyzer.cs : DiagnosticAnalyzer
```